src/swap/components/SwapButton.tsx

```diff
@@ -18,6 +20,17 @@ export function SwapButton({ className, disabled = false }: SwapButtonReact) {
     disabled ||
     isLoading;
 
+  // prompt user to connect wallet
+  if (!isDisabled && !address) {
+    return (
+      <Wallet className="mt-4 w-full">
+        <ConnectWallet className="w-full">
+          <Avatar className="h-6 w-6" />
```

Comment:
Do we need `<Avatar className="h-6 w-6" />` and `<Name />`, I don't think so, right?

Reply:
+1, I think we just need [useConnect](https://wagmi.sh/react/api/hooks/useConnect)

Reply:
done ! using `ConnectWallet` component